lldb/source/Plugins/OperatingSystem/SwiftTasks/OperatingSystemSwiftTasks.h

```diff
@@ -49,10 +50,18 @@ class OperatingSystemSwiftTasks : public OperatingSystem {
   /// If a thread for task_id had been created in the last stop, return it.
   /// Otherwise, create a new MemoryThread for it.
   lldb::ThreadSP FindOrCreateSwiftThread(ThreadList &old_thread_list,
-                                         uint64_t task_id);
+                                         uint64_t task_id,
+                                         std::optional<std::string> task_name);
 
-  /// Find the Task ID of the task being executed by `thread`, if any.
-  std::optional<uint64_t> FindTaskId(Thread &thread);
+  /// Find the address of the Task being executed by `thread`, if any.
+  std::optional<lldb::addr_t> FindTaskAddress(Thread &thread);
+
+  /// Find the ID of the Task at the given address, if any.
+  std::optional<uint64_t> FindTaskId(lldb::addr_t task_addr);
```

Comment:
can you add `Process&` paremeter here and move this method as a non-member in the cpp file?

Reply:
This one is still here!

Reply:
This was not introduced in this PR (though its signature has changed). It also refers `m_job_id_offset` and so I kept it a member function, as is.

Reply:
I'll rewrite it, but that's why I had left it a member function.

Reply:
Ohhh, we can do it later, it's fine

Reply:
done